GitHubTeams.ps1

```diff
@@ -276,6 +315,488 @@ filter Get-GitHubTeamMember
+        if ($PSBoundParameters.ContainsKey('RepositoryFullName')) { $hashBody['repo_names'] = $RepositoryFullName }
+        if ($PSBoundParameters.ContainsKey('Privacy')) { $hashBody['privacy'] = $Privacy.ToLower() }
+        if ($MaintainerName.Count -gt 0)
+        {
+            $hashBody['maintainers'] = $maintainerNames
+        }
+        if ($PSBoundParameters.ContainsKey('ParentTeamName'))
+        {
+            $getGitHubTeamParms = @{
+                OrganizationName = $OrganizationName
+                TeamName = $ParentTeamName
+                Whatif = $false
+                Confirm = $false
+            }
+            if ($PSBoundParameters.ContainsKey('AccessToken'))
+            {
+                $getGitHubTeamParms['AccessToken'] = $AccessToken
+            }
+            if ($PSBoundParameters.ContainsKey('NoStatus'))
+            {
+                $getGitHubTeamParms['NoStatus'] = $NoStatus
+            }
+
+            $team = Get-GitHubTeam @getGitHubTeamParms
+
+            $hashBody['parent_team_id'] = $team.id
+        }
+
+        $params = @{
+            UriFragment = $uriFragment
+            Body = (ConvertTo-Json -InputObject $hashBody)
+            Method = 'Post'
+            Description =  "Creating $TeamName"
+            AccessToken = $AccessToken
+            TelemetryEventName = $MyInvocation.MyCommand.Name
+            TelemetryProperties = $telemetryProperties
+            NoStatus = (Resolve-ParameterWithDefaultConfigurationValue `
+                -Name NoStatus -ConfigValueName DefaultNoStatus)
+        }
+
+        return (Invoke-GHRestMethod @params | Add-GitHubTeamAdditionalProperties)
+    }
+}
+
+filter Set-GitHubTeam
+{
+<#
+    .SYNOPSIS
+        Updates a team within an organization on GitHub.
+
+    .DESCRIPTION
+        Updates a team within an organization on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OrganizationName
+        The name of the team's organization.
+
+    .PARAMETER TeamName
+        The name of the team.
+
+    .PARAMETER Description
+        The description for the team.
+
+    .PARAMETER Privacy
+        The level of privacy this team should have.
+
+    .PARAMETER ParentTeamName
+        The name of a team to set as the parent team.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .INPUTS
```

Comment:
Looks like `GitHub.Organization` is a valid input as well.

Reply:
Done